include/GL/glui.h

```diff
@@ -1587,40 +1654,39 @@ class GLUIAPI GLUI_EditText : public GLUI_Control
     int                 text_x_offset;
     int                 substring_start; /*substring that gets displayed in box*/
     int                 substring_end;  
-    int                 sel_start, sel_end;  /* current selection */
-    int                 num_periods;
```

Comment:
I've done more hands on with GLUI this week (I'm on vacation) than in the past 10 years.  It's not good form to insult folks for asking questions, or insulting previous contributors.  For what it's worth I have studied this convoluted branch at great length, as a courtesy.